hazimp/jobs/vulnerability_model.py

```diff
@@ -45,13 +45,13 @@ def vuln_sets_from_xml_file(filename):
     :returns: A dictionary of Vulnerability Sets.
     """
 
-    # TODO: no XSD validation is currently performed. This requires
-    # lxml, which is not available in GA's runtime environment
-    # currently. Also needs an XSD for for NRML data.
-
     if not os.path.exists(filename):
         raise RuntimeError("No vulnerability XML was loaded." +
                            " Check file name " + filename)
+
+    validator = Validator(NRLM_SCHEMA)
+    validator.validate(filename)

```

Comment:
This will currently just propagate the AssertionError from lxml, any preference on whether this should be wrapped with our own exception / handled with a more user friendly message @wcarthur?

Reply:
At this point in time, I am comfortable to allow the AssertionError to propagate. If there's ongoing feedback about the error messages, we'll address in the future